### PR TITLE
Add docs for MaxConcurrentStacks configuration

### PR DESCRIPTION
**Pull request by zaro0508**

### CODE REVIEW

File: docs/task-files.md [hunk 1]
```diff
@@ -304,6 +305,7 @@ The ``update-serverless.com`` task will deploy the [serverless.com](https://serv
 |Skip| `true` or `false` |When `true` task (and dependent tasks) will not be executed.|
 |TaskRoleName|string|Specifies the name of the IAM Role that must be used for cross account access. A role with this is expected to exist in the target account (and have the right AssumeRole permissions).|
 |Parameters|any|Specifies parameters that must be passed to the serverless deployment using command arguments.|
+|MaxConcurrentStacks|number|The number of stacks that should be executed concurrently.<br/><br/> Default = 1|
```
Comment (OlafConijn) [inline docs/task-files.md:308]: For udpate-serverless and update-cdk i believe the setting is called MaxConcurrentTasks.
Also: the setting, currently, only allows for its default value of 1. not sure whether it makes sense to have these attributes listed

File: docs/task-files.md [hunk 2]
```diff
@@ -250,6 +250,7 @@ The `update-stacks` task will provision all resources in all accounts specified
 |OrganizationBindings|Dictionary of String, [OrganizationBinding](#organizationbinding-where-to-create-which-resource)| Set of named OrganizationBindings that can be `!Ref`'d by Resources.<br/><br/> **note**: This value overrides values within the template or resources (value in taskfile is leading).|
 |CloudFormationRoleName|string|Specifies the name of the IAM Role that must be used to pass to the CloudFormation service. A role with this is expected to exist in the target account (and have the right AssumeRole permissions).|
 |TaskRoleName|string|Specifies the name of the IAM Role that must be used for cross account access. A role with this is expected to exist in the target account (and have the right AssumeRole permissions).|
+|MaxConcurrentStacks|number|The number of stacks that should be executed concurrently.<br/><br/> Default = 1|
```
Comment (OlafConijn) [inline docs/task-files.md:253]: great! thanks!

**Review by OlafConijn — CHANGES_REQUESTED** [review]

File: docs/task-files.md (hunk 1 above)
Comment (zaro0508) [reply docs/task-files.md:308]: > For udpate-serverless and update-cdk i believe the setting is called MaxConcurrentTasks.

Hmm,  the example just below has `MaxConcurrentStacks`?  

> Also: the setting, currently, only allows for its default value of 1. not sure whether it makes sense to have these attributes listed

Do you mean the only value that you can set for this is 1?  Does this same limitation also apply to `update-cdk`?

**Comment by OlafConijn** [comment]
fixed the docs in this PR: https://github.com/org-formation/org-formation-cli/pull/222
thanks! @zaro0508

**Pull request closed without merging**